src/app/components/users-container/users-list/users-list.component.ts

```diff
@@ -1,16 +1,142 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, SimpleChanges, OnChanges } from '@angular/core';
 import { User } from 'ngx-login-client';
-
+import { ListConfig } from 'patternfly-ng';
+import { Filter } from 'patternfly-ng';
+import { FilterConfig } from 'patternfly-ng';
+import { FilterField } from 'patternfly-ng';
+import { FilterEvent } from 'patternfly-ng';
+import { FilterType } from 'patternfly-ng';
+import { SortConfig } from 'patternfly-ng';
+import { SortField } from 'patternfly-ng';
+import { SortEvent } from 'patternfly-ng';
+import { ToolbarConfig } from 'patternfly-ng';
 @Component({
   selector: 'app-users-list',
   templateUrl: './users-list.component.html',
   styleUrls: ['./users-list.component.css']
 })
-export class UsersListComponent implements OnInit {
+export class UsersListComponent implements OnInit, OnChanges {
 
   @Input() users: User[];
 
-  ngOnInit() {
+  listConfig: ListConfig;
+  filterConfig: FilterConfig;
+  filtersText: String = '';
+  items: any[];
```

Comment:
Why is items type `any[]`? It should be `User[]`.

Reply:
Forgot!.Will Fix it!.